src/libs/utils.js

```diff
@@ -186,10 +186,12 @@ export const generateClusterName = () => `saturn-${uuid()}`
 
 export const waitOneTick = () => new Promise(setImmediate)
 
-export const entityAttributeText = value => {
+export const entityAttributeText = (value, machineReadable) => {
```

Comment:
So, since the limitations around this are known, I'm not going to break this up for now. Definitely a further conversation to be had once it's possible to define links/lists in a flat file, though.